.github/workflows/update-libs.yaml

```diff
@@ -0,0 +1,46 @@
+name: Auto-update Charm Libraries
```

Comment:
Lets drop the workflow entirely rather than commenting most of it

Reply:
I kept it there as a TODO thing so that we do not forget.